lib/openfoodfacts.dart

```diff
@@ -77,16 +78,20 @@ class OpenFoodAPIClient {
   ///
   /// Please read the language mechanics explanation if you intend to display
   /// or update data in specific language: https://github.com/openfoodfacts/openfoodfacts-dart/blob/master/DOCUMENTATION.md#about-languages-mechanics
-  static Future<Status> saveProduct(User user, Product product,
-      {QueryType queryType = QueryType.PROD}) async {
+  static Future<Status> saveProduct(
+    User user,
+    Product product, {
+    QueryType queryType = QueryType.PROD,
+  }) async {
     var parameterMap = <String, String>{};
     parameterMap.addAll(user.toData());
     parameterMap.addAll(product.toServerData());
 
-    var productUri = Uri(
-        scheme: URI_SCHEME,
-        host: queryType == QueryType.PROD ? URI_PROD_HOST : URI_TEST_HOST,
-        path: '/cgi/product_jqm2.pl');
+    var productUri = Settings.getUri(
```

Comment:
You should deprecate `URI_SCHEME`, `URI_PROD_HOST`, `URI_TEST_HOST`, `URI_PROD_HOST_ROBOTOFF` and `URI_TEST_HOST_ROBOTOFF`.

Reply:
Ohh thanks for the reminder, missed that one